gedcom/families.py

```diff
@@ -294,3 +296,27 @@ def _validate_marr_div_dates(self, family):
                                        "Married date should occur before current date for a family")
             return False
         return True
+
+    def _validate_parents_not_too_old(self, family):
+        children = family.get_children()
+
+        for child in children:
+            child = self._people.individuals[child]  # type: Person
+            husband = self._people.individuals[family.get_husband_id()]  # type: Person
+            wife = self._people.individuals[family.get_wife_id()]  # type: Person
+            if husband.get_age() - child.get_age() >= 80:
+                self._msgs.add_message(self.CLASS_IDENTIFIER,
+                                       "US12",
+                                       family.get_family_id(),
+                                       "NA",
+                                       "Father should be less than 80 years older than his children")
+                return False
```

Comment:
Don't need to return false here as nothing uses this return value.  For testing you should be using .validate() and checking the messages returned as this is a private method.